### ISSUE TITLE
1.12.2zb Missing recipe for the seeds of the new crops. 

### ISSUE THREAD

**Issue by Pablin345** [MatrexsVigil/harvestcraft#313]
The new Crops: Agave, Amaranth, Arrowroot, Cassava, Chickpea, Elderberry, Flax, Green Grape, Huckleberry, Jicama, Jute, Kenaf, Kohlrabi, Lentil, Millet, Mulberry, Quinoa, Sisal, and Taro.

They do not have the recipe for crafting seed from the drop of the gardens,

None of the new crops, has the recipe.

**Comment by luke-bravenboer** [comment]
Can confirm that the seeds for the aforementioned crops have no crafting recipe, as well as Kale seeds too. They're still obtainable in the market, you just can't craft these crops into their seed directly

**Comment by luke-bravenboer** [comment]
Here's my own Crafttweaker workaround for now, feel free to use it in the meantime:

```
recipes.addShapeless(<harvestcraft:kaleseeditem>, [<harvestcraft:kaleitem>]);
recipes.addShapeless(<harvestcraft:agaveseeditem>, [<harvestcraft:agaveitem>]);
recipes.addShapeless(<harvestcraft:amaranthseeditem>, [<harvestcraft:amaranthitem>]);
recipes.addShapeless(<harvestcraft:arrowrootseeditem>, [<harvestcraft:arrowrootitem>]);
recipes.addShapeless(<harvestcraft:cassavaseeditem>, [<harvestcraft:cassavaitem>]);
recipes.addShapeless(<harvestcraft:chickpeaseeditem>, [<harvestcraft:chickpeaitem>]);
recipes.addShapeless(<harvestcraft:elderberryseeditem>, [<harvestcraft:elderberryitem>]);
recipes.addShapeless(<harvestcraft:flaxseeditem>, [<harvestcraft:flaxitem>]);
recipes.addShapeless(<harvestcraft:greengrapeseeditem>, [<harvestcraft:greengrapeitem>]);
recipes.addShapeless(<harvestcraft:huckleberryseeditem>, [<harvestcraft:huckleberryitem>]);
recipes.addShapeless(<harvestcraft:jicamaseeditem>, [<harvestcraft:jicamaitem>]);
recipes.addShapeless(<harvestcraft:juteseeditem>, [<harvestcraft:juteitem>]);
recipes.addShapeless(<harvestcraft:kenafseeditem>, [<harvestcraft:kenafitem>]);
recipes.addShapeless(<harvestcraft:kohlrabiseeditem>, [<harvestcraft:kohlrabiitem>]);
recipes.addShapeless(<harvestcraft:lentilseeditem>, [<harvestcraft:lentilitem>]);
recipes.addShapeless(<harvestcraft:milletseeditem>, [<harvestcraft:milletitem>]);
recipes.addShapeless(<harvestcraft:mulberryseeditem>, [<harvestcraft:mulberryitem>]);
recipes.addShapeless(<harvestcraft:quinoaseeditem>, [<harvestcraft:quinoaitem>]);
recipes.addShapeless(<harvestcraft:sisalseeditem>, [<harvestcraft:sisalitem>]);
recipes.addShapeless(<harvestcraft:taroseeditem>, [<harvestcraft:taroitem>]);
```

**Comment by MatrexsVigil** [comment]
Fixed in 1.12.2zc
